src/statistic/update.py

```diff
@@ -48,21 +48,6 @@ def update_all_stats(self):
 
```

Comment:
Update script failed on first execution. Since updating some of the analysis  results, the script works fine.